packages/store/src/StoreConsumer.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { IStoreConsumer } from "./IStore.sol";
+
+/**
+ * Default Store consumers consider their caller to be the Store, thus delegating access control to the Store.
+ * The Store contract considers itself to be the Store (so StoreRead has a different storeAddress definition).
+ * Tests/scripts may override this value for convenience.
+ * WARNING: World Systems rely on storeAddress being msg.sender for access control!
```

Comment:
Not sure if we should put this warning here - the Store package should be independent of the World package, and the warning itself is not detailed enough to be helpful without the full context. Maybe we should a more detailed explanation in the docs and link it somewhere in the world package.

Reply:
added a comment to `WorldConsumer` instead
also agree with docs, should it be in this PR or separate?